@xen-orchestra/web/src/components/tree/HostTreeItem.vue

```diff
@@ -2,8 +2,16 @@
   <TreeItem :expanded="branch.isExpanded">
     <TreeItemLabel :icon="faServer" :route="`/host/${branch.data.id}`" @toggle="branch.toggleExpand()">
       {{ branch.data.name_label }}
+      <template #icon>
+        <ObjectIcon
+          v-tooltip="branch.data.power_state"
```

Comment:
IMO, the `branch.data.power_state` tooltip should also be in the `VmTreeItem`. Maybe that already plan in another PR?